src/index.js

```diff
@@ -0,0 +1,177 @@
+  let kelvin;
+  axios
+    .get('http://127.0.0.1:5000/weather', {
+      params: { lat: state.lat, lon: state.long },
+    })
+    .then((response) => {
+      kelvin = response.data['main']['temp'];
+      state.temp = Math.floor(getFarenheit(kelvin));
+      tempGroundColorChange();
+    })
+    .catch((error) => {
+      console.log(`error in getting weather of ${state.city}`);
+    });
+};
+
+const tempUp = () => {
+  state.temp += 1;
+  // console.log(state.temp);
+  tempGroundColorChange();
+};
+
+const tempDown = () => {
+  state.temp -= 1;
+  // console.log(state.temp);
+  tempGroundColorChange();
+};
+
+const tempGroundColorChange = () => {
+  let temp = state.temp;
+  // console.log(temp);
+  let color = 'green';
+  let landscape = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+
+  if (temp >= 80) {
+    color = 'red';
+    landscape = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (70 <= temp && temp <= 79) {
+    color = 'orange';
+    landscape = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (60 <= temp && temp <= 69) {
+    color = 'brown';
+    landscape = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (50 <= temp && temp <= 59) {
+    color = 'green';
+    landscape = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else {
+    color = 'purple';
+    landscape = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+
+  const landscapeDisplay = document.getElementById('ground-display');
+  landscapeDisplay.textContent = landscape;
+  const displayTemp = document.getElementById('temp');
+  displayTemp.style.color = color;
+  displayTemp.textContent = String(state.temp);
+};
+
+const resetName = () => {
+  let cityReset = document.getElementById("city-name-input");
+  cityReset.value = "Seattle";
+  changeName();
+
+  // state.city = 'Seattle';
+  // let name = document.getElementById('city-name-display');
+  // name.textContent = state.city;
+  // let newName = document.getElementById('city-name-input');
+  // newName.value = state.city;
+};
+
+const changeName = () => {
+  let name = document.getElementById('city-name-display');
+  let newName = document.getElementById('city-name-input'); //newName = inputName
+  
+  state.city = newName.value;
+  name.textContent = state.city;
+};
+
+const changeSky = () => {
+  const skyDisplay = document.getElementById('sky-display');
+  const skyBackground = document.getElementById('landscape-display');
+  const skyChoice = document.getElementById('sky-choice').value;
+  let skyEmoji;
+  let skyColor;
+
+  // console.log(skyDisplay);
+  // console.log(skyBackground);
+  // console.log(skyChoice);
+
+  if (skyChoice === 'Sunny') {
+    skyEmoji = '☁️ ☁️ ☀️ ☁️ ☁️';
+    skyColor = '#cbedfb';
+  } else if (skyChoice === 'Cloudy') {
+    skyEmoji = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+    skyColor = '#b0e0e6';
+  } else if (skyChoice === 'Rainy') {
+    skyEmoji = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+    skyColor = '#B0C4DE';
+  } else if (skyChoice === 'Snowy') {
+    skyEmoji = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+    skyColor = '#F0F8FF';
+  }
+
+  skyDisplay.textContent = skyEmoji;
+  skyBackground.style.backgroundColor = skyColor;
```

Comment:
Consider using classes instead of setting the color directly. This allows us to have all the styling information the the CSS file, and allows us easily have other type of styling be impacted by the temperature (for example if we decided we wanted the font to change depending on the temperature as well).